overlord/devicestate/devicemgr.go

```diff
@@ -181,6 +181,7 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 	runner.AddHandler("request-serial", m.doRequestSerial, nil)
 	runner.AddHandler("mark-preseeded", m.doMarkPreseeded, nil)
 	runner.AddHandler("mark-seeded", m.doMarkSeeded, nil)
+	runner.AddHandler("setup-ubuntu-save", m.doPrepareUbuntuSave, nil)
```

Comment:
the method name should match the task name: doSetupUbuntuSave

Reply:
nice! Renamed this, and also renamed the prepareUbuntuSave I had used in unit tests.